src/commands/step_processor.go

```diff
@@ -85,16 +87,18 @@ func (s *stepSession) processSingleSession() {
 	result, err := s.Client().Installer.GetNextSteps(s.Context(), &params)
 	if err != nil {
 		s.Logger().Warnf("Could not query next steps: %s", err.Error())
+		return defaultNextStepOnErrorTimeout
```

Comment:
Why? If there was error and we wait 120 seconds, the bm-inventory will move the host to disconnected state.  I don't think there should be backoff here

Reply:
I agree with @ori-amizur 
Unless we got the next value from BM-inventory, we should just use the default